Web/starter_project_barbies/pages/story/index.tsx

```diff
@@ -0,0 +1,20 @@
+import Title from '../../components/successStory/Title'
+import  Profile  from '../../components/successStory/SuccessStory'
+import Partners from '../../components/successStory/Partners'
+import Footer from '../../components/successStory/Footer'
+import { Inter } from 'next/font/google'
+
+
+const inter = Inter({ subsets: ['latin'] })
+
+export default function Home() {
+  return (
+    <div>
+      <Title />
+      <Profile />
+      <Partners />
+      <Footer />
```

Comment:
This should be definef in the layout. It shouldn't be included in the Home components.

Reply:
done